rules.d/meson.build

```diff
@@ -12,6 +12,7 @@ rules = [
                '60-drm.rules',
                '60-evdev.rules',
                '60-fido-id.rules',
+               '60-gpiochip.rules',
```

Comment:
Hmm, is a new udev rules file something that should be added in stable? This is really a new feature, not a bugfix.

Reply:
Yeah, I don't think we should introduce new udev rules in backports, unless they are targeted fixes at a bug.